baby-liminal-extension/src/backend/benchmarking.rs

```diff
@@ -57,12 +99,18 @@ mod benchmarks {
         }
     }
```

Comment:
Do we also include the `.encode()` part in the benchmark?

Reply:
encoding happens in the contract